woodwork/column_accessor.py

```diff
@@ -85,13 +88,58 @@ def __eq__(self, other):
             return False
         return self._series.equals(other._series)
 
+    def __getattr__(self, attr):
+        '''
+            If the method is present on the Accessor, uses that method.
+            If the method is present on DataFrame, uses that method.
+        '''
+        if self._schema is None:
+            raise AttributeError("Woodwork not initialized for this Series. Initialize by calling Series.ww.init")
+        if hasattr(self._series, attr):
+            return self._make_series_call(attr)
+        else:
+            raise AttributeError(f"Woodwork has no attribute '{attr}'")
+
     def __repr__(self):
         msg = u"<Series: {} ".format(self._series.name)
         msg += u"(Physical Type = {}) ".format(self._series.dtype)
         msg += u"(Logical Type = {}) ".format(self.logical_type)
         msg += u"(Semantic Tags = {})>".format(self.semantic_tags)
         return msg
 
+    def _make_series_call(self, attr):
+        '''
+        Forwards the requested attribute onto the series object.
+        Intercepts return value, attempting to initialize Woodwork with the current schema
+        when a new Series is returned.
+        Confirms schema is still valid for the original Series.
+        '''
+        series_attr = getattr(self._series, attr)
+
+        if callable(series_attr):
+            def wrapper(*args, **kwargs):
+                # Make Series call and intercept the result
+                result = series_attr(*args, **kwargs)
+
+                # Try to initialize Woodwork with the existing Schema
+                if isinstance(result, pd.Series):
+                    if result.dtype == self._schema['logical_type'].pandas_dtype:
+                        # We don't need to pass dtype from the schema to init
+                        schema = {key: value for key, value in self._schema.items() if key != 'dtype'}
+                        result.ww.init(**schema)
```

Comment:
This'll create a new object for the column dictionary, but it won't make a deepcopy. 

We can wait to handle this in #582 if we want, but I just wanted to point out that with this implementation, changes to `metadata` and `semantic_tags` would happen to both schemas. (The same issue exists with the Table Schema, but worse because the Schema is the exact same object at the moment)

Reply:
Ah, good point.

Reply:
I went ahead and added a deepcopy operation here to resolve this issue.

Reply:
Awesome! I'll add a note in 582 so that we remember to test this, but glad we've got it here!